src/musicPlaylist.java

```diff
@@ -0,0 +1,21 @@
+public interface musicPlaylist {
```

Comment:
This isn't quite right. Is this the kernel? The other file is called kernel but inherits from kernel. It's a bit of a mess.